tests/python/gaia-ui-tests/gaiatest/apps/search/app.py

```diff
@@ -10,6 +10,7 @@
 class Search(Base):
 
     name = 'Browser'
+    origin_app_name = 'search'
```

Comment:
I think this is still 'browser', actually.
However, this stuff doesn't seem to be used at all, currently.